components/post_view/post_list_virtualized/post_list_virtualized.test.jsx

```diff
@@ -177,6 +177,11 @@ describe('PostList', () => {
                 scrollOffset: 490,
                 expected: true,
             },
+            {
+                name: 'when  pixel from the bottom also considered to be bottom',
```

Comment:
This test does not seem to add any value here as it is already. covered in the above assertion. Could you remove it?

Reply:
resolved it